src/main/java/org/folio/service/orders/PurchaseOrderLineService.java

```diff
@@ -112,14 +114,20 @@ public Future<Void> saveOrderLine(CompositePoLine compositePoLine, RequestContex
     return saveOrderLine(poLine, requestContext);
   }
 
+  public Future<Void> saveOrderLines(PoLineCollection poLineCollection, RequestContext requestContext) {
+    RequestEntry requestEntry = new RequestEntry(BATCH_ENDPOINT);
+    return restClient.put(requestEntry, poLineCollection, requestContext);
+  }
+
 
   public Future<Void> saveOrderLines(List<PoLine> orderLines, RequestContext requestContext) {
-    return GenericCompositeFuture.join(orderLines.stream()
-      .map(poLine -> saveOrderLine(poLine, requestContext)
-        .recover(t -> {
-          //TODO: always POL_LINES_LIMIT_EXCEEDED message hides other error messages. to fix
-          throw new HttpException(400, ErrorCodes.POL_LINES_LIMIT_EXCEEDED.toError());
-        }))
+    List<PoLineCollection> poLineCollections = ListUtils.partition(orderLines, 100)
```

Comment:
You added this 100 constant because it is used also in re-encumber logic to not have big payloads?
Because for rollover batching already is using

Reply:
Yeah, I know we're sending 200 POLs to the rollover. Honestly, I don't know the maximum allowed server size for a POST request, so I decided to be on the safe side.

Reply:
Okay, lets introduce constant for this, or better to use the same as we doing for rollover.

Reply:
done